src/resources/views/crud/buttons/quick.blade.php

```diff
@@ -23,6 +23,24 @@
         $wrapper['href'] = ($wrapper['href'])($entry, $crud);
     }
     $wrapper['class'] = $wrapper['class'] ?? $defaultClass;
+    //if ajax enabled
+    $ajax_enabled = $button->meta['ajax'] ?? false;
```

Comment:
My Initial thought:
If so, why are we not doing the same for existing operation buttons?
Let's say if we do for all - But no. of file will increase.

**Then my brain argues**, They are not two-in-one buttons like this. If they are the Ajax buttons, they are only Ajax buttons, so the **files will not increase**.

My final thought is that I like to keep this tiny thing in a single file; I don't like jumping on files for this small thing.

So it's a preference thing. Let me know your final command; I will walk in your direction, Master :)

Reply:
I usually agree with that, but in this case, and like you pointed out, this represent two buttons in one. 
It's not that we have plans to re-use the ajax part (theoretically we could) but this 40 line file, went to 100 lines, so more than half of it is now the ajax functionality.

In this case I still think it's better to move the ajax part to a separate file, it will help to keep both files clean and easy to understand. 

We can check later with Cristian about his opinion on this, it's not something that's preventing us from moving forward, if we decide to do it, there shouldn't be any code changes needed, so we leave this to decide at the end 👍